tsfile/src/main/java/org/apache/iotdb/tsfile/common/conf/TSFileConfig.java

```diff
@@ -1,187 +1,186 @@
-
-  // Don't change the following configuration
-  /**
-   * Default block size of two-diff. delta encoding is 128
-   */
-  public static int deltaBlockSize = 128;
-  /**
-   * Default frequency type is SINGLE_FREQ.
-   */
-  public static String freqType = "SINGLE_FREQ";
-  /**
-   * Default PLA max error is 100.
-   */
-  public static double plaMaxError = 100;
-  /**
-   * Default SDT max error is 100.
-   */
-  public static double sdtMaxError = 100;
-  /**
-   * Default DFT satisfy rate is 0.1
-   */
-  public static double dftSatisfyRate = 0.1;
-  /**
-   * Data compression method, TsFile supports UNCOMPRESSED or SNAPPY. Default value is UNCOMPRESSED
-   * which means no compression
-   */
-  public static String compressor = "UNCOMPRESSED";
-  /**
-   * Line count threshold for checking page memory occupied size.
-   */
-  public static int pageCheckSizeThreshold = 100;
-  /**
-   * Default endian value is BIG_ENDIAN.
-   */
-  public static String endian = "BIG_ENDIAN";
-
-  /**
-   * Default storage is in local file system
-   */
-  public static FSType TSFileStorageFs = FSType.LOCAL;
-
-  /**
-   * Default hdfs ip is localhost
-   */
-  public static String hdfsIp = "localhost";
-
-  /**
-   * Default hdfs port is 9000
-   */
-  public static String hdfsPort = "9000";
-
-  public TSFileConfig() {
-
-  }
-
-
-  public static FSType getTSFileStorageFs() {
-    return TSFileStorageFs;
-  }
-
-  public static void setTSFileStorageFs(String TSFileStorageFs) {
-    TSFileConfig.TSFileStorageFs = FSType.valueOf(TSFileStorageFs);
-  }
-
-  public static String getHdfsIp() {
-    return hdfsIp;
-  }
-
-  public static void setHdfsIp(String hdfsIp) {
-    TSFileConfig.hdfsIp = hdfsIp;
-  }
-
-  public static String getHdfsPort() {
-    return hdfsPort;
-  }
-
-  public static void setHdfsPort(String hdfsPort) {
-    TSFileConfig.hdfsPort = hdfsPort;
-  }
-}
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.iotdb.tsfile.common.conf;
+
+import org.apache.iotdb.tsfile.fileSystem.FSType;
+
+/**
+ * TSFileConfig is a configure class. Every variables is public and has default value.
+ *
+ * @author kangrong
+ */
+public class TSFileConfig {
```

Comment:
why this file is changed totally...

Reply:
@jixuan1989 Already fixed it. I merge all commits into one commit.